manual/modules/ROOT/pages/coding-guidelines.adoc

```diff
@@ -77,6 +77,24 @@ results in the http://opencpn.github.io/OpenCPN/api-docs/classAbstractRestServer
 . When working with existing code, gradually refactor to comply with coding guidelines.
 . Do not submit large PRs that refactor a lot of code changes.
 
+=== Localization and Internationalization
```

Comment:
I'd be more than happy to adjust the guidelines based on what's practical. As a new contributor, I'm not sure what the best practices are; it helps to write the guidelines explicitly. For example, who is supposed to run `make pot-update` and `make po-update`?

Reply:
As said: For release of OCPN Dave, bdbcat. use to maintain localization. And also handle updates from Crowdin when translators are ready. So not much to describe in a developer manual for OCPN code I guess.
I sometimes run the pot creation if I've edited text and like to check it. But not to be included in any PR.
For external plugins the author have to take care often with the help from Nohal or me.

Reply:
> So not much to describe in a developer manual for OCPN code I guess.

Based on that, I have updated to:

```doxygen
. Maintainers run the following commands before each release. Do not run them in your pull requests.
.. `make pot-update` - Scans source files and updates the template (`.pot`) file.
.. `make po-update` - Updates all language files (`.po`) with new strings from template, to merge new strings into existing language files.
```